src/app/codegen-data-model/tests/TestCodegenModelViaMocks.cpp

```diff
@@ -341,6 +353,10 @@ const MockNodeConfig gTestNodeConfig({
             MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_IPV6ADR_ATTRIBUTE_TYPE),
             MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_IPV6PRE_ATTRIBUTE_TYPE),
             MOCK_ATTRIBUTE_CONFIG_NULLABLE(ZCL_HWADR_ATTRIBUTE_TYPE),
+
+            // Special case handling
+            MockAttributeConfig(kAttributeIdReadOnly, ZCL_INT32S_ATTRIBUTE_TYPE, 0),
```

Comment:
Comment about this file --> this is not a Mock, it's a fake. Don't need to change it in this PR, but I think it's worth calling it a fake later.

Reply:
I inherited the naming from existing tests and the name "mock" propagated throughout.